cmd/beacon-chain/main.go

```diff
@@ -288,7 +288,7 @@ func startNode(ctx *cli.Context) error {
 
 	beacon, err := node.New(ctx, opts...)
 	if err != nil {
-		return err
+		return fmt.Errorf("unable to start beacon node: %w", err)
```

Comment:
This will break wrapping, would it be better to use errors.Wrap here?

Reply:
@kasey I don't think it breaks wrapping. You can unwrap an error that was templated with `%w`. https://pkg.go.dev/errors#example-Unwrap

Reply:
Oh my mistake - TIL the ` %w` verb, very cool!